docs/topics/getting-started-intro.adoc

```diff
@@ -1,4 +1,8 @@
-Developing cloud native applications and services is different than developing on-premise, monolithic applications. When learning how to create cloud native applications and services, developers need a local environment that matches the target cloud environment. The {launcher} application provides a painless way for developers to create functional, example applications called missions and easily build and deploy them to a cloud environment. The missions are working applications that showcase different fundamental pieces of building cloud native applications and services, such as creating HTTP APIs, interoperating with a database, or implementing the health check pattern. Each mission is comprised of different boosters that show how to implement the same mission in different runtimes and can be updated or extended for your own use case. Using {launcher} with the missions keeps you focused on learning to develop applications and services for the target cloud environment from the beginning.
+When starting to create cloud-native applications and services, you need an local development environment that matches the target cloud environment.
+The {OpenShiftLocal} provides an out-of-the-box solution for setting up a cloud environment on your local machine without the need for extensive setup and configuration, while the {launcher} application gives you a hassle-free way of creating functional example applications called Missions and easily building and deploying them to a cloud environment.
+The Missions are designed to showcase different fundamental pieces of building cloud-native applications and services, such as creating HTTP APIs, integrating you applications with a relational database to provide persistent data storage, implementing the health check and circuit-breaker patterns to ensure that your services can withstand overload and network issues, externalizing the configuration of your applications to make them more secure and easier to scale, and securing your applications with Red Hat Single Sign-on to provide authentication and authorization functionalities.
+Each mission comprises different boosters that show how to implement the same mission in different runtimes and can be updated or extended for your own use case.
```

Comment:
Maybe capitalize _Booster_, like Mission was capitalized above? To show that it's a new term.